src/compiler/scala/tools/reflect/FastTrack.scala

```diff
@@ -25,6 +25,8 @@ class FastTrack[MacrosAndAnalyzer <: Macros with Analyzer](val macros: MacrosAnd
     new { val c: c0.type = c0 } with Taggers
   private implicit def context2macroimplementations(c0: MacroContext): FormatInterpolator { val c: c0.type } =
     new { val c: c0.type = c0 } with FormatInterpolator
+  private implicit def context2macroStringImplementations(c0: MacroContext): StringInterpolator { val c: c0.type } =
```

Comment:
not sure if this change should modify the FormatInterpolator, or be standlone

It does share some behaviour